### ISSUE TITLE
Navigating date selector via keyboard

### ISSUE THREAD

**Issue by lizscuffell** [epimorphics/ppd-explorer#122]
Ensure the date selector can be navigated via keyboard easily. It's not that intuitive. You have to press function+up/down to change the month, but it's not obvious as to how to change the day as function+left/right moves the page content up and down

**Comment by ijdickinson** [comment]
@lizscuffell Should be fixed now

**Comment by lizscuffell** [comment]
Date selector functionality is now navigable. Closing ticket
